aspnetcore/performance/response-compression.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Response Compression Middleware | Microsoft Docs
+author: guardrex
+description: An introduction to response compression with instructions on how to use Response Compression Middleware in ASP.NET Core applications.
+keywords: ASP.NET Core, performance, response compression, gzip, accept-encoding, middleware
+ms.author: riande
+manager: wpickett
+ms.date: 01/12/2017
+ms.topic: article
+ms.assetid: de621887-c5c9-4ac8-9efd-f5cc0457a134
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: performance/response-compression
+---
+# Response Compression Middleware
+
+By [Luke Latham](https://github.com/GuardRex)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample)
+
+This document introduces response compression and explains how to use ASP.NET Core Response Compression Middleware. A response compression sample application demonstrates the middleware and response compression concepts.
+
+You can explore the features of the Response Compression Middleware with the [sample application](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample). The sample illustrates the compression of application responses using GZip and custom compression providers. It also shows you how to add a MIME type to the default list of MIME types for compression.
```

Comment:
nit: on the GNU site and in RFC 7230 it's stylized Gzip not GZip. But I've also seen GZIP and gzip elsewhere, not sure which one is better.